webapp/components/Registration/CreateUserAccount.vue

```diff
@@ -1,78 +1,88 @@
 <template>
-  <ds-card v-if="success" class="success">
-    <ds-space>
-      <sweetalert-icon icon="success" />
-      <ds-text align="center" bold color="success">
-        {{ $t('registration.create-user-account.success') }}
-      </ds-text>
-    </ds-space>
-  </ds-card>
-  <ds-form
-    v-else
-    class="create-user-account"
-    v-model="formData"
-    :schema="formSchema"
-    @submit="submit"
-  >
-    <template slot-scope="{ errors }">
-      <ds-card :header="$t('registration.create-user-account.title')">
-        <ds-input
-          id="name"
-          model="name"
-          icon="user"
-          :label="$t('settings.data.labelName')"
-          :placeholder="$t('settings.data.namePlaceholder')"
-        />
-        <ds-input
-          id="bio"
-          model="about"
-          type="textarea"
-          rows="3"
-          :label="$t('settings.data.labelBio')"
-          :placeholder="$t('settings.data.labelBio')"
-        />
-        <ds-input
-          id="password"
-          model="password"
-          type="password"
-          autocomplete="off"
-          :label="$t('settings.security.change-password.label-new-password')"
-        />
-        <ds-input
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-08-14T09:39:15Z" title="Wednesday, August 14th 2019, 11:39:15 am +02:00">Aug 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -109,30 +112,38 @@ export default {
       disabled: true,
       success: null,
       backendErrors: null,
+      checkedDefault: false,
     }
   },
   props: {
     nonce: { type: String, required: true },
     email: { type: String, required: true },
   },
   methods: {
+    checked: function() {
+      this.backendErrors = { message: '' }
+    },
```


Nice! I like this. :smile:

Why not set `backendErrors` back to `null`?

And I would find it easier to read if the function was called `resetErrors` – because then I could see right in the template what happens `@change`.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-08-14T10:28:10Z" title="Wednesday, August 14th 2019, 12:28:10 pm +02:00">Aug 14, 2019</time>_

---


I changed it after your suggestion.